testing/test_data.py

```diff
@@ -341,5 +342,35 @@ def setUp(self):
         self.ref_data_index = os.path.join(testing_root_dir,
             'references', 'data', 'testcase3', 'tc3_data_retrieved_index.csv')
 
+class FindDaysTest(unittest.TestCase, utilities.partialChecks):
+    '''Tests module to find peak and typical heating and cooling
+    days for a test case. These days are used to define the
+    test case scenarios.
+
+    '''
+
+    def setUp(self):
+        '''Setup for each test.
+
+        '''
+
+        self.sim_data = os.path.join(testing_root_dir,'references',
+                            'data', 'find_days', 'sim_test_days.csv')
+
+        self.days_ref = os.path.join(testing_root_dir,'references',
+                            'data', 'find_days', 'days_ref.json')
+
+
+    def test_find_days(self):
+        '''The test uses one month simulation data as obtained from
+        the bestest_air case.
+
+        '''
+
+        days = find_days(heat='fcu_reaPCoo_y', cool='fcu_reaPHea_y',
```

Comment:
I suggest testing with using thermal load instead of electrical or gas power.

Reply:
Addressed in https://github.com/ibpsa/project1-boptest/pull/285/commits/a0e7d9e84d91714e6b561aa9682424ab11f83fb5.